playbooks/roles/ocp-cost-management/tasks/cmo-cleanup.yml

```diff
@@ -1,120 +1,66 @@
 ---
-# this file is used for cleaning up the compliance operator resources
-
-- name: Check if ScanSettingBinding exists
-  shell: oc get ScanSettingBinding -n costmanagement-metrics-operator --no-headers | awk '{print $1}'
-  register: scan_setting_binding
-
-- name: Delete the ScanSettingBinding if exists
-  kubernetes.core.k8s:
-    api_version: compliance.openshift.io/v1alpha1
-    state: absent
-    kind: ScanSettingBinding
-    name: "{{ item }}"
-    namespace: costmanagement-metrics-operator
-  loop: "{{ scan_setting_binding.stdout_lines }}"
-  when: scan_setting_binding.stdout_lines|length > 0
-
-- name: Check if ScanSetting exists
-  shell: oc get ScanSetting -n costmanagement-metrics-operator --no-headers | awk '{print $1}'
-  register: scan_setting
-
-- name: Delete the ScanSetting if exists
-  kubernetes.core.k8s:
-    api_version: compliance.openshift.io/v1alpha1
-    state: absent
-    kind: ScanSetting
-    name: "{{ item }}"
-    namespace: costmanagement-metrics-operator
-  loop: "{{ scan_setting.stdout_lines }}"
-  when: scan_setting.stdout_lines|length > 0
-
-- name: Check if ComplianceSuite exists
-  shell: oc get ComplianceSuite -n costmanagement-metrics-operator --no-headers | awk '{print $1}'
-  register: compliance_suite
-
-- name: Delete the ComplianceSuite if exists
-  kubernetes.core.k8s:
-    api_version: compliance.openshift.io/v1alpha1
-    state: absent
-    kind: ComplianceSuite
-    name: "{{ item }}"
-    namespace: costmanagement-metrics-operator
-  loop: "{{ compliance_suite.stdout_lines }}"
-  when: compliance_suite.stdout_lines|length > 0
+# this file is used for cleaning up the cost management operator resources
 
 - name: Get the name of cluster service version for Compliance Operator
   shell: oc get csv -n costmanagement-metrics-operator --no-headers | awk '{ if ($1 ~ /^compliance-operator/) print $1 }'

```

Comment:
Check for cost management operator instead of compliance-operator.

Reply:
done